core/src/main/kotlin/me/lazmaid/kraph/Kraph.kt

```diff
@@ -11,22 +11,29 @@ typealias FieldBlock = Kraph.FieldBuilder.() -> Unit
 typealias CursorBlock = Kraph.CursorSelectionBuilder.() -> Unit
 typealias NodeBlock = Kraph.NodeBuilder.() -> Unit
 
+data class KraphVariableType(val value: String)
+data class KraphVariable(val name: String, val type: KraphVariableType, val jsonValue: String) {
```

Comment:
I think these 2 classes deserve their own package. Maybe we can have the package for custom GraphQL type. What do you think?

Reply:
You mean a package that has a class with only those 2 lines of code?
Or any other class as well?

Reply:
For now, It would be another package with a file contain these 2 classes